### DISCUSSION TITLE
Customize output of annotated notes

### DISCUSSION THREAD

**Discussion by adibMosharrof** [argenos/zotero-mdnotes#97] [Q&A]
In this [video](https://www.youtube.com/watch?v=XbGJH08ZfCs&t=1103s), the output format of the extracted notes have been customized. Could you please share how we can do this?

**Comment by argenos** [comment]
Have you checked the documentation? https://argentinaos.com/zotero-mdnotes/docs/next/advanced/formatting

**Reply by adibMosharrof** [reply]
I have checked that documentation. Unfortunately, I did not understand how to do the custom formatting by reading that page.

**Reply by argenos** [reply]
What have you tried so far?

**Reply by adibMosharrof** [reply]
I have changed the different predefined templates defined [here](https://argentinaos.com/zotero-mdnotes/docs/next/advanced/templates/defaults)

But what I found is that they dont actually change the note content. 

In the link that you provided, it talks about formatting rules, but I really didnt understand what is going on in that page and how the different things are operating. It would be great if you could direct me to some resources that might help me to understand what is going on, and do the custom formatting that I initially planned on.

**Comment by argenos** [comment]
In that case, please take a look at the docs about templates: https://argenos.github.io//zotero-mdnotes/docs/next/advanced/templates/defaults 

Please note that the templates you need to edit also depend on your configuration, so it might be good to check the “getting started” section as well: https://argenos.github.io//zotero-mdnotes/docs/next/getting-started/configuration 

Hope that helps!
